oaknational/Oak-Web-Application
# LESQ-767: Update copyright year published lessons

## Description

Music year: 2004

- Updates year of new published lessons and test files

## Issue(s)

Fixes #LESQ-767

## Acceptance Criteria:

- [ ]  Update licence date on new lessons depending on when they were last published (both lesson overview and downloads)
- [ ]  Legacy content licence copy should remain unchanged

## How to test

1. Go to https://deploy-preview-2396--oak-web-application.netlify.thenational.academy

New lessons should have same year as the updated_at from internal releases table date on both lessons and downloads pages

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2024-04-29 at 15 09 31](https://github.com/oaknational/Oak-Web-Application/assets/91190841/19bd256d-bef3-4099-9341-8afafbd2aa1a)

How it should now look:
![Screenshot 2024-04-29 at 15 09 09](https://github.com/oaknational/Oak-Web-Application/assets/91190841/da77b54f-e328-47fa-a55d-dce1c5fa8afa)
